### PR TITLE
Implement implicit null insert.

### PR DESCRIPTION
**Pull request by boomkim**
Regarding issue #197 . 

Now it checks `column_def` is nullable and then if it's nullable and have no value(`None`), it returns `Expr::Literal(Null)`

### CODE REVIEW

**Review by ever0de — APPROVED** [review]
very good change
Waiting for the maintainer's review.

File: src/data/row.rs
Comment (ever0de) [inline src/data/row.rs:85]: Q: How about moving this function into `try_into_value`?

**Comment by panarch** [comment]
fix #197

**Review by panarch — APPROVED** [review]
Looks great :)
Nice to see `match` code updates.
Thanks! merging :)

**Pull request merged**
